### ISSUE TITLE
Unable to send emails from a Debian 11 server

### ISSUE THREAD

**Issue by WilliamBehrens** [LukeSmithxyz/emailwiz#200]
Hello, I've setup a Debian 11 vps on Linode and I used this script to setup, after trying to send emails a tutanota account, a gmail account and trying to use a [dkim test](https://www.appmaildev.com/en/dkim) none of my emails went through. I am able to receive emails just fine(I'm using geary and k9 mail) but just can't seem to send any. Is there any additional I need to provide?

**Comment by Lukasz825700516** [comment]
After fresh install on debian11 neomutt rises `SASL authentication failed` while sending mail.

journalctl:
```
Feb 07 11:21:25 localhost postfix/smtpd[20217]: warning: [ip stuff]: SASL LOGIN authentication failed: UGFzc3dvcmQ6
Feb 07 11:21:25 localhost postfix/smtpd[20217]: lost connection after AUTH from [ip stuff]
Feb 07 11:21:25 localhost postfix/smtpd[20217]: disconnect from [ip stuff] ehlo=2 starttls=1 auth=0/1 commands=3/4
```

**Comment by WilliamBehrens** [comment]
Any update on this or what I can do to fix it? should I just delete my current email and start a new debian 10 server? can I backup everything from my current one?

**Comment by ThamognyaKodi** [comment]
Fr @WilliamBehrens 
This emailwiz script works on my Debian 11 VPS. Maybe try opening port 25, 587, and 993 in ufw or your firewall tool and ask for an opening of port 25 in vultr. That might solve it or you can continue this issue. 


For @Lukasz825700516 
> After fresh install on debian11 neomutt rises `SASL authentication failed` while sending mail.
> 
> journalctl:
> 
> ```
> Feb 07 11:21:25 localhost postfix/smtpd[20217]: warning: [ip stuff]: SASL LOGIN authentication failed: UGFzc3dvcmQ6
> Feb 07 11:21:25 localhost postfix/smtpd[20217]: lost connection after AUTH from [ip stuff]
> Feb 07 11:21:25 localhost postfix/smtpd[20217]: disconnect from [ip stuff] ehlo=2 starttls=1 auth=0/1 commands=3/4
> ```

That is more of a neomutt issue than a issue with the script. If you used lukesmith's muttwizard for neomutt, post an issue in https://github.com/LukeSmithxyz/mutt-wizard

**Comment by WilliamBehrens** [comment]
How would I do that? I'm using linode, not vultr

**Comment by ChemicalXandco** [comment]
> I'm using linode, not vultr

Have you opened a support ticket to unblock ports 25, 465, and 587? https://www.linode.com/docs/guides/running-a-mail-server/

**Comment by michael-keyes** [comment]
Same issues here. Ports 25, 465, 587, 993 are open. Debian 11 server. Not using mutt

**Comment by ThamognyaKodi** [comment]
I have gone back and tried this on vultr, and SSDnodes Debian 11, apache and nginx and it seems to be working. I know this statement is not helpful, but I will try to break it down into a few bullet points which you can check to see if you have made a mistake assuming you did not tweak the script:

- [ ] check for mx record
- [ ] check for cname record
- [ ] check for reverse dns
- [ ] make a dummy website and certbot it
- [ ] add new users with mail group and /bin/bash with a `useradd -m -G mail -s /bin/bash _user_name_` as without the **-s /bin/bash**, the new user won't have a shell as I noticed. (Although it should not really change the outcome, I am just adding this since I did it)
- [ ] If you are using a firewall, open port 25, 587, 993. For ufw it will be `ufw allow 25` , `ufw allow 587`, `ufw allow 993`
- [ ] ask for a opening of port 25 (not needed for SSDNodes)

I also suggest you to watch Lukesmith's video here: https://youtu.be/9zP7qooM4pY as it lists everything in a simpler format.

I know, I have only tested this with 2 providers, but this check list should work with most providers (including linode and etc.)
